pkg/reconciler/stats_reporter.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconciler
+
+import (
+	"context"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"knative.dev/pkg/metrics"
+)
+
+const (
+	knativeServingEditCountName = "knativeserving_edit_count"
+)
+
+var (
+	knativeServingEditCountStat = stats.Int64(
+		knativeServingEditCountName,
+		"Number of KnativeServing changes",
```

Comment:
Could you list off the type of edits being included in this count in the description? i.e. create, delete, modify, etc.

[nit] Use "edits" instead of "changes" for consistency.